cfpb/college-costs
# Add verify info interactions with no JS

Added no-JS friendly dynamic interaction for settlement students to verify their information, or signify that it isn't their information.
## Additions
- Added `div` wrappers for `info-right` and `info-wrong`
- Added `target` selectors to use when showing and hiding verification instructions
## Changes
- Changed HTML verification `buttons` to `a` elements, so the `target` attribute could be used.
- Moved instructions for what to do if a student receives the wrong offer information lower in the HTML code
## Testing
- Pull in the `no-js` branch and run `gulp`. Load it up, and try saying that it isn't your offer info, or that it is.
- Try it in standalone or in UnityBox. Try it on a phone or tablet.
- Try it in a browser with Javascript disabled, and see that it works there.
- Try it in IE8, to see that **it does not work there**. Luckily, the JS enhancement should, unless our IE8 user has also disabled JavaScript.

I tested it on Mac Chrome 47, Mac Safari 9.0, Mac Safari 9.0 with JS disabled, IE8, and Android Chrome Beta 48 on a phone.
## Review
- @niqjohnson @ascott1 @mistergone @higs4281 
## Todos
- Add JS to hide the button/link and some content
- Add keyframes animation for a smoother interaction
- Work with the other FEWD-like folks to see if we can restructure the page to have the Big Question buttons act the same way. I don't see a good way to do it currently without a more major restructuring of the page.
## Screenshots

On load:

![screen shot 2015-12-29 at 4 39 22 pm](https://cloud.githubusercontent.com/assets/1183435/12042862/1efaddd0-ae4d-11e5-9c29-41aa154bb5dc.png)
## Checklist
